data/initrd/scripts/remote_log_setup

```diff
@@ -0,0 +1,66 @@
+#! /bin/sh
+#USAGE: ./script_name $remote_log_host_ip
+
+#preconditions
```

Comment:
please do not patch our own system; add the link directly to the initrd via data/initrd/initrd.file_list